app/src/components/Panels/NewRequest.js

```diff
@@ -65,12 +65,14 @@ const NewRequest = React.memo(({ panelOpened, acceptedTokens, onRequest }) => {
   const [orgToken, setOrgToken] = useState(initialState.orgToken)
 
   useEffect(() => {
-    if (selectedToken.index === -1) {
-      setSelectedToken({
-        ...initialState.selectedToken,
-        index: acceptedTokens.length > 0 ? 0 : initialState.selectedToken.index,
-        value: acceptedTokens.length > 0 ? acceptedTokens[0].address : initialState.selectedToken.value,
-      })
+    if (acceptedTokens && acceptedTokens.length > 0) {
+      if (selectedToken.index === -1) {
+        setSelectedToken({
+          ...initialState.selectedToken,
+          index: acceptedTokens.length > 0 ? 0 : initialState.selectedToken.index,
+          value: acceptedTokens.length > 0 ? acceptedTokens[0].address : initialState.selectedToken.value,
```

Comment:
It's not necessary to check the conditions here. Should return the first value since now we are checking for the condition at the top